Resources/Prototypes/Entities/Structures/Dispensers/chem.yml

```
@@ -7,8 +7,10 @@
   components:
   - type: Sprite
     sprite: Structures/dispensers.rsi
-    state: industrial-working
-    snapCardinals: true
+# ES Start
+    state: industrial
+    norot: True
```

```suggestion
    noRot: True
```
epic webedit numero uno